hapi/hedera-protobufs/services/state/network/network.proto

```diff
@@ -0,0 +1,84 @@
+/**
+ * # Network
+ * A representation that includes all “biographical information” on each
+ * node that is needed by a network with TSS fully operational.
+ * Required in order to replace the address book with Rosters.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+
+syntax = "proto3";
+
+package com.hedera.hapi.node.state.network;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import "auxiliary/tss/tss_message.proto";
+import "state/addressbook/node.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.network">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+
+
+/**
+ * The full information needed to bootstrap a network with TSS enabled
+ * and the legacy address book completely replaced by rosters.
```

Comment:
Style is off here.  We're falling back into "describe" habit rather than "specify" habit.
In addition to _adding_ requirements below this, this description should be reworded
```suggestion
 * A full description of the Network of Consensus Nodes.<br/>
 * This records the full information needed to bootstrap a network
 * with TSS enabled and the legacy address book completely replaced
 * by rosters.
 *
 * TBD Need requirements detailed here, including block stream effects.
```